kolibri/core/assets/src/views/k-modal.vue

```diff
@@ -0,0 +1,312 @@
+<template>
+
+  <!-- Accessibility properties for the overlay -->
+  <transition name="fade">
+    <div
+      class="modal-overlay"
+      @keyup.esc="emitCancelEvent"
```

Comment:
Replaced keydown with keyup. More standard.

Reply:
I don't actually believe this is true?